usr/local/share/deskpro/container-var-reference.json

```diff
@@ -1,419 +1,711 @@
+    "type": "integer",
+    "default": "1"
+  },
+  {
+    "name": "SVC_EMAIL_PROCESS_ARGS_EXTRA",
+    "description": "Additional arguments to the email processing service.",
+    "type": "string"
   },
   {
-    "description": "Dont perform email collection/processing as part of cron (i.e. indicates that email services are being used instead)",
-    "name": "TASKS_DISABLE_EMAIL_IN_JOB"
+    "name": "SVC_EMAIL_PROCESS_ARGS_MAX_JOBS",
+    "description": "Stop processing after this many messages or 0 to have no limit.",
+    "type": "integer",
+    "default": "300"
+  },
+  {
+    "name": "SVC_EMAIL_PROCESS_ARGS_MAX_TIME",
+    "description": "The maximum number of seconds the email processing command can run before exiting or 0 to have no max time.",
+    "type": "integer",
+    "default": "600"
+  },
+  {
+    "name": "SVC_EMAIL_PROCESS_ARGS_RESERVE_TIME",
+    "description": "The number of email messages to reserve at one time.",
+    "type": "integer",
+    "default": "5"
+  },
+  {
+    "name": "SVC_EMAIL_PROCESS_ARGS_TIMEOUT",
+    "description": "Max time to spend on any single message before timing out.",
+    "type": "integer",
+    "default": "300"
+  },
+  {
+    "name": "SVC_EMAIL_PROCESS_ARGS_TRIES",
+    "description": "Number of times to attempt a processing a message before logging it failed.",
+    "type": "integer",
+    "default": "3"
+  },
+  {
+    "name": "SVC_EMAIL_PROCESS_ENABLED",
+    "description": "Set in 15-run-mode.sh and used in supervisor config - determines if email processing service should run as a standalone process.",
+    "type": "boolean",
+    "default": "false",
+    "isPrivate": true
+  },
+  {
+    "name": "SVC_EMAIL_PROCESS_NUMPROCS",
+    "description": "Number of processes to run for email processing.",
+    "type": "integer",
+    "default": "1"
+  },
+  {
+    "name": "SVC_NGINX_ENABLED",
+    "description": "Set in 15-run-mode.sh and used in supervisor config - determines if nginx service should run.",
+    "type": "boolean",
+    "default": "false",
+    "isPrivate": true
+  },
+  {
+    "name": "SVC_NGINX_LISTEN_ADDRESS",
+    "description": "The 'listen' directive for the nginx server block.",
+    "type": "string",
+    "default": "*"
+  },
+  {
+    "name": "SVC_PHP_FPM_ENABLED",
+    "description": "Set in 15-run-mode.sh and used in supervisor config - determines if PHP FPM service should run.",
+    "type": "boolean",
+    "default": "false",
+    "isPrivate": true
+  },
+  {
+    "name": "SVC_TASKS_ENABLED",
+    "description": "Set in 15-run-mode.sh and used in supervisor config - determines if tasks service should run.",
+    "type": "boolean",
+    "default": "false",
+    "isPrivate": true
+  },
+  {
+    "name": "TASKS_DISABLE_EMAIL_IN_JOB",
+    "description": "Set in 15-run-mode.sh and used in Deskpro config - don't perform email collection/processing as part of cron (i.e. indicates that email services are being used instead)",
```

Comment:
Description isn't quite right in reference to it being set in 15-run-mode.sh.

This is a user variable that should be set if you enable the separate email processing services (i.e. when SVC_EMAIL_COLLECT_ENABLED/SVC_EMAIL_PROCESS_ENABLED=true). It prevents email processing from being duplicated in normal cron runs.

It gets automatically set as a convenience if you have 1 node running both cron+email services. But if you ran 2 nodes instead (i.e. 1 for cron, 1 for email), then you'd need to make sure to set it manually to prevent cron from running email tasks.